fitbenchmarking/fitbenchmarking
# 219 move across wiki

#### Description of Work

Fixes #219 

Information on the wiki has been moved across to sphinx.  Note: information has been moved across as-is, and may still be out of date

#### Testing Instructions

1. run `make html` in the docs directory
2. check that outputted files are formatted correctly, links work, image display, etc.
3. check that there's nothing in the wiki that's missing from the sphinx docs
